modules/javafx.graphics/src/main/java/com/sun/glass/events/KeyEvent.java

```diff
@@ -252,11 +252,16 @@ public class KeyEvent {
      * Returns a VK_ code of a key capable of producing the given unicode
      * character with respect to the currently active keyboard layout or
      * VK_UNDEFINED if the character isn't present in the current layout.
+     * The hint is the KeyCode of the key the system is attempting to match.
+     * It can be used to optimize the search or to distinguish between the
+     * main keyboard and the numeric keypad.
```

Comment:
should we mention that it is supposed to be `KeyEvent.VK_UNDEFINED` when not available?

Reply:
I updated the comment here as well.